LCN: migrate actions to dedicated pages (#46153)
home-assistant/home-assistant.github.io: source/_actions/lcn.var_abs.markdown

---
title: "Set absolute variable"
action: lcn.var_abs
domain: lcn
description: "Sets absolute value of a variable or setpoint."
related_actions:
  - lcn.var_rel
  - lcn.var_reset
  - lcn.lock_regulator
---

> The **Set absolute variable** action sets a variable or setpoint on an LCN module to a fixed value. Use it to write a known value, for example a target temperature, to a regulator setpoint.

Reviewer: ```suggestion
This action sets a variable or setpoint on an LCN module to a fixed value. Use it to write a known value, for example a target temperature, to a regulator setpoint.
```

### Options in the UI

{% options_ui %}
Device:
  description: The LCN module or group to send the command to.
  required: true
Variable:
  description: The variable or setpoint name.
  required: true
Value:
  description: The value to set.
  required: false
Unit of measurement:
  description: The unit of the value. Defaults to `native`.
  required: false
{% endoptions_ui %}

{% include actions/yaml_header.md %}

In YAML, refer to this action as `lcn.var_abs`. A basic example looks like this:

{% example %}
action: |
  action: lcn.var_abs
  data:
    device_id: 91aa039a2fb6e0b9f9ec7eb219a6b7d2
    variable: var1
    value: 75
    unit_of_measurement: "%"
{% endexample %}

This sets variable 1 of the selected module to 75%.

### Options in YAML

{% options_yaml %}
device_id:
  description: >
    The device ID of the LCN module or group.
  required: true
  type: string
variable:
  description: >
    The variable or setpoint name. See
    [Variables and units](/integrations/lcn/#variables-and-units).
  required: true
  type: string
value:
  description: >
    The value to set. If not provided, it is assumed to be 0.
  required: false
  default: 0

>   type: integer

Author: It is a float in the schema.

unit_of_measurement:
  description: >
    The unit of the value. See
    [Variables and units](/integrations/lcn/#variables-and-units). If not
    provided, it is assumed to be `native`.
  required: false
  type: string
{% endoptions_yaml %}

{% important %}
Ensure that the LCN module is configured properly to provide access to the defined variable. Otherwise the module might show unexpected behavior or return error messages.
{% endimportant %}

{% include actions/try_it.md %}

{% include actions/more_examples.md %}

> 

Reviewer: ```suggestion
```
